tests/reg_tests/test_euler_tutorial_wing.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import print_function
+import unittest
+import numpy
+from baseclasses import BaseRegTest
+import os
+import sys
+baseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(baseDir,'../../'))
+from python.pyADflow import ADFLOW
+import copy
+from baseclasses import AeroProblem
+# from reg_tests.commonUtils import standard_test, adflowDefOpts, defaultFuncList
+from commonUtils import *    
+from runpy import run_path
+
+
+class TestSolution(unittest.TestCase):
```

Comment:
It would be useful to have a more descriptive name here.

Reply:
How about TestSolve? 
That test checks to make sure adflow can still solve the case to the same tolerance and that the solved state and functions are consistent with the reference.
Ideally we'd added more detail to the test description too.

Reply:
Yes agreed. I saw that there's a standard_test, is that basically the same thing as what's done here?

Reply:
This specific test would be analogous to solution_test in the old framework.
TestFuntionals performs the same tests as standard_test